src/main.c

```diff
@@ -209,25 +209,31 @@ int main(int argc, char **args)
     run_flag |= opt_prof_data << 2;
 
     vm_attr_t attr = {
-        .mem_size = MEM_SIZE,
+#if RV32_HAS(SYSTEM) && !defined(USE_ELF)
```

Comment:
Is it making sense to have both `SYSTEM` and `USE_ELF` at the same time?

Reply:
Specify `SYSTEM`  for binding MMU callback and specify `USE_ELF` indicating that the input file is ELF. This is essential when running MMU test suite.